test_scripts/API/AddCommand/ATF_AddCommand_UNSUP_RES_to_UI_AddCmd_WARNINGS_to_VR_AddCmd.lua

```diff
@@ -0,0 +1,134 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
```

Comment:
@istoimenova , Please see updated in commit e3c195283c2acd34494b065de5dde8270b56f351